staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/noconversion.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conversion
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// NoConversionConverter is a converter that does only set the apiVersion and kind, but without actual conversions.
+type NoConversionConverter struct {
```

Comment:
would merge those two converters for now. We can add abstractions when we need another instance for declarative conversion.

Reply:
I changed the structure of converters with some chat comments from @liggitt look at them again.